docs/skillmap/educator-info/bug-arena-info.md

```diff
@@ -0,0 +1,129 @@
+- Explain how AI systems perform tasks using precise language that avoids anthropomorphism
+
+**Designing AI**
+- Describe how AI systems can be designed to support a solution to a community problem
+- Evaluate AI systems using defined criteria, expected outcomes, and user feedback
+- Describe an AI model's purpose, intended users, and its limitations
+
+## Lesson Materials
+
+Materials available to use for this lesson include:
+- Educator Guide
+- PowerPoint Presentation with talking points
+- Student worksheet to print
+- Student worksheet example solution
+- Certificate to print
+
+All materials are posted at: [https://aka.ms/bug-arena-educator](https://aka.ms/bug-arena-educator) 
+
+## Code definitions
+
+### On Start
+
+![On Start](/static/skillmap/bug-arena/onstart.png "on start")
+
+The code inside the **On Start** block will run as soon as the program starts.
+
+### Every
+
+![Every](/static/skillmap/bug-arena/every.png "every")
+
+The code inside the **Every** block will run on a specified millisecond (ms) time interval.
+
+### On Bump Wall
+
+![On Bump Wall](/static/skillmap/bug-arena/onbumpwall.png "on bump wall")
+
+The code inside the **On Bump Wall** block will run when the bug bumps into a wall.
+
+### Turn
+
+![Turn](/static/skillmap/bug-arena/turn.png "turn")
+
+Will turn the bug a specified number of degrees from its current direction. A positive number is clockwise, a negative number is counter-clockwise.
+
+![Turn angle](/static/skillmap/bug-arena/turn-angle.png "turn angle")
+
+### Face Towards
+
+![Face Towards](/static/skillmap/bug-arena/facetowards.png "face towards")
+
+Will turn the bug to face a specific angle in degrees where: 0 = right, 90 = down, 180 = left, 270 = up
+
+![Face towards degrees](/static/skillmap/bug-arena/degrees.png "face towards degrees")
+
+### Run After
+
+![Run After](/static/skillmap/bug-arena/runafter.png "run after")
+
+Will wait a specified amount of time in milliseconds (ms) before running the code inside it.
+
+### Distance to Color
+
+![Distance to Color](/static/skillmap/bug-arena/distancetocolor.png "distance to color")
+
+Will return the distance in pixels from the front of the bug to its own paint color.
+
+### Pick Random
+
+![Pick Random](/static/skillmap/bug-arena/pickrandom.png "pick random")
+
+Will return a random whole number between a minimum and maximum value.
+
+## Game Mod Ideas
+
+Once students have completed the Bug Arena skillmap, they can modify (mod) their programs using the full MakeCode editor. Use the **[SAVE TO MY PROJECTS]** button in the Skillmap to continue editing the code. 
+
+Mod Suggestions: 
+
+- Add music or sound effects
+- Add screen effects
+- Try using different algorithm strategies to beat the Tower
+
+## What's Next?
+
+When students are finished with **Bug Arena** consider encouraging them to complete another skillmap.
+
+- [Try the Jungle Monkey Skillmap](/skillmap/jungle)
```

Comment:
Reference the page view of these or use launch links?

Reply:
Sorry, what's the difference?

Reply:
It loads a markdown render of the schema instead of going to the skillmap. I think we need to change `/skillmap/jungle` to `https://arcade.makecode.com/--skillmap#docs:/skillmap/jungle`

Reply:
Ah, okay.  Let's do that then.  Thanks!